### PR TITLE
@craigspaeth Fixes mis-interpreted URLs and rips out Artist orchestration layer

### PR DESCRIPTION
**Pull request by kanaabe**
Closes https://github.com/artsy/positron/issues/341
Goodbye artist orchestration layer, now using v1 only
Fixes small bug with loading spinner on featured artists/artworks section

### CODE REVIEW

**Comment by craigspaeth** [comment]
Wohoo! Very nice! Thanks!

File: client/apps/edit/components/admin/test/index.coffee [hunk 1]
```diff
@@ -64,7 +64,7 @@ describe 'EditAdmin', ->
       @view.featureFromInput('Artists')(
         currentTarget: $ "<input value='/andy-warhol' />"
       )
-      Backbone.sync.args[0][2].data.ids[0].should.equal 'andy-warhol'
+      Backbone.sync.args[0][1].id.should.equal 'andy-warhol'
```
Comment (kanaabe) [inline client/apps/edit/components/admin/test/index.coffee]: We aren't using the `ApiCollection` anymore for artist so I'm just testing against the request's `id` param

Comment (craigspaeth) [inline client/apps/edit/components/admin/test/index.coffee]: :+1: works for me.

File: client/collections/artworks.coffee [hunk 2]
```diff
@@ -18,6 +18,7 @@ module.exports = class Artworks extends Backbone.Collection
       new Artwork(id: id).fetch
         error: options.error
         success: (artwork) -> cb null, artwork
+        complete: options.complete
     , (err, artworks) =>
```
Comment (kanaabe) [inline client/collections/artworks.coffee]: The `complete` method was missing here which allows the `featureFromInput` in the mentioned/featured artworks section to be called back. Was previously holding on to the loading spinner.

Comment (craigspaeth) [inline client/collections/artworks.coffee]: Ah ha! Good catch!

**Pull request merged**